skimage/color/colorconv.py

```diff
@@ -94,15 +94,16 @@ def guess_spatial_dimensions(image):
 def convert_colorspace(arr, fromspace, tospace):
     """Convert an image array to a new color space.
 
+    Valid color spaces include:
+        'RGB', 'HSV', 'RGB CIE', 'XYZ', 'YUV', 'YIQ', 'YPbPr', 'YCbCr', 'YDbDr'
```

Comment:
include -> are

wrap this line to 80 chars since we aren't in a specially-formatted section now

Reply:
I'll have you know, that it is exactly 79 characters long ;)
![image](https://user-images.githubusercontent.com/90008/40214861-fdd2118c-5a11-11e8-9717-6c26e387f7db.png)

Reply:
Touché!!! LOL!